app/app_test.go

```diff
@@ -56,8 +69,14 @@ func (at *appTest) reset() {
 	at.accOut = types.MakeAcc("output0")
 
 	eyesCli := eyes.NewLocalClient("", 0)
-	at.app = NewBasecoin(eyesCli)
-	at.app.SetLogger(log.TestingLogger().With("module", "app"))
+	// l := log.TestingLogger().With("module", "app"),
+	l := log.NewTMLogger(os.Stdout).With("module", "app")
```

Comment:
okay. do it. i should review the coding standards.

I find `cpStateBytes` a ridiculously long variable name though, and go style does encourage short-ish names.